great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -81,7 +81,6 @@
 if TYPE_CHECKING:
     from sqlalchemy.sql import quoted_name  # noqa: TID251 # type-checking only
 
-    from great_expectations.compatibility import sqlalchemy
```

Comment:
We currently  import sqlalchemy in the non-typing section aliased to `sa` so removing this is cleanup.

Reply:
See comments added to code. I no longer remove this line.